doc/nrf/ug_thread_ot_integration.rst

```diff
@@ -14,26 +14,34 @@ Overview
 ********
 
 The OpenThread stack is integrated with the Zephyr RTOS through the L2 layer.
+See :ref:`zephyr:network_stack_architecture` for more information.
 
-From the architecture perspective, the L2 layer is using the nRF IEEE 802.15.4 radio driver, which is located underneath the L2 layer.
-It can also optionally use Zephyr's IPv6 stack, which is located above it.
-The OpenThread API and IPv6 stack can be used directly, but by default, Zephyr's IPv6 stack is used.
-This has the following advantages and disadvantages:
+The OpenThread integration in the L2 layer uses the nRF variant of the :ref:`zephyr:ieee802154_interface` radio driver, which is located underneath the L2 layer.
+It can also optionally use Zephyr's IP stack, which is located above it.
 
-* The advantage of using Zephyr's IPv6 stack is that it can use Zephyr's L3 layer, which allows for application portability and usage of UDP, TCP, and ICMP built on top of Zephyr's IPv6 stack.
-* The disadvantage is that the reception path where IP packets need to traverse the OpenThread L3 layer and then Zephyr's L3 layer in order to reach the BSD socket.
+Alternatively, you can use the OpenThread API and IPv6 stack directly.
```

Comment:
Feel free to ignore this if no simple solution: I wonder if this sentence should be a little more discouraging from this approach. In the future architectures this API may not be available on application core this can make porting applications harder.

Reply:
Added a suggestion, please check.